python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -534,6 +509,24 @@ def run(self, inputExp, inputDims, taskMetadata, inputPhotodiodeData=None):
                 im1Area, im2Area, imStatsCtrl, mu1, mu2 = self.getImageAreasMasksStats(exp1, exp2,
                                                                                        region=region)
 
+                # Get the read noise for each exposure
+                readNoise1 = dict()
+                readNoise2 = dict()
+                meanReadNoise = dict()
+
+                expMetadata1 = expRef1.get(component="metadata")
+                metadataIndex1 = inputDims.index(expId1)
+                thisTaskMetadata1 = taskMetadata[metadataIndex1]
+
+                expMetadata2 = expRef2.get(component="metadata")
+                metadataIndex2 = inputDims.index(expId2)
+                thisTaskMetadata2 = taskMetadata[metadataIndex2]
+
+                readNoise1[ampName] = self.getReadNoise(expMetadata1, thisTaskMetadata1, ampName)
+                readNoise2[ampName] = self.getReadNoise(expMetadata2, thisTaskMetadata2, ampName)
+
+                meanReadNoise[ampName] = np.nanmean([readNoise1[ampName], readNoise2[ampName]])
```

Comment:
Initially I didn't understand why you need this change but now I probably do.  
Since you want to save each read noise at each flux level, you only have two measurement at each flux. Then there is no way to take median. That's why you need to switch to `nanmean`?

Reply:
Yep!